lib/help-search.js

```diff
@@ -26,28 +27,17 @@ class HelpSearch extends BaseCommand {
 
   async helpSearch (args) {
     if (!args.length)
-      throw this.usage
+      return this.npm.output(this.usage)
 
     const docPath = path.resolve(__dirname, '..', 'docs/content')
-
     const files = await glob(`${docPath}/*/*.md`)
     const data = await this.readFiles(files)
     const results = await this.searchFiles(args, data, files)
-    // if only one result, then just show that help section.
-    if (results.length === 1) {
-      return this.npm.commands.help([path.basename(results[0].file, '.md')], er => {
-        if (er)
-          throw er
-      })
-    }
-
     const formatted = this.formatResults(args, results)
     if (!formatted.trim())
-      npmUsage(this.npm, false)
-    else {
+      this.npm.output(`No matches in help for: ${args.join(' ')}\n`)
```

Comment:
Now that this isn't the usage output for the cli itself we can act in isolation and not have to return usage here.